src/native/libs/System.Security.Cryptography.Native/pal_evp_pkey.c

```diff
@@ -109,6 +109,37 @@ int32_t CryptoNative_EvpPKeyType(EVP_PKEY* key)
     }
 }
 
+int32_t CryptoNative_EvpPKeyFamily(const EVP_PKEY* key)
```

Comment:
Originally, I thought about trying to re-purpose `CryptoNative_EvpPKeyType` to not return the actual base ID, but instead just return an enum that we map (just like we do here). The problem with that is we really need the actual base ID from `CryptoNative_EvpPKeyType` because it passes the ID back to OpenSSL in some cases. We can't just arbitrarily make up our own base IDs, so this became a separate function.